rviz_default_plugins/src/rviz_default_plugins/displays/marker/markers/triangle_list_marker.cpp

```diff
@@ -55,6 +62,39 @@ namespace displays
 namespace markers
 {
 
+resource_retriever::MemoryResource getResource(const std::string & resource_path)
```

Comment:
Presuming this is a local function:
```suggestion
static resource_retriever::MemoryResource getResource(const std::string & resource_path)
```